cmd/inetdata-csvrollup/main.go

```diff
@@ -94,6 +95,18 @@ func mergeAndEmit(c chan OutputKey, o chan string) {
 	wg.Done()
 }
 
+func allowedIdentical(lookup string) bool {
+	switch lookup {
+	case
+		"ns",
+		"r-ns",
```

Comment:
Including `r-ns` since this can now appear after the changes to `inetdata-sonardnsv2-split`